### PR TITLE
Crowdstrike falcon x

### PR DESCRIPTION
**Pull request by merit-maita**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/47218

## Description
added args to cs-fx-upload-file command.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py [hunk 1]
```diff
@@ -739,10 +739,18 @@ def test_module(client: Client) -> str:
...
         is_confidential: str = "true",
         comment: str = "",
         submit_file: str = "no",
+        action_script: str = "",
+        command_line: str = "",
+        document_password: str = "",
+        enable_tor: str = "false",
+        submit_name: str = "",
+        system_date: str = "",
+        system_time: str = ""
```
Comment (yaakovi) [inline Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py:753]:
```suggestion
        system_time: str = "",
```

**Review by yaakovi — CHANGES_REQUESTED** [review]
update the detonation playbooks as well

**Comment by ShirleyDenkberg** [comment]
@yaakovi Doc review completed.

File: Packs/CrowdStrikeFalconX/Playbooks/playbook-FalconX_Detonate_and_Analyze_File.yml [hunk 2]
```diff
@@ -318,6 +318,16 @@ tasks:
...
+      enable_tor:
+        simple: "false"
+      environment_id:
+        simple: '160: Windows 10'
+      action_script: {}
+      command_line: {}
+      document_password: {}
+      submit_name: {}
+      system_date: {}
+      system_time: {}
```
Comment (yaakovi) [inline Packs/CrowdStrikeFalconX/Playbooks/playbook-FalconX_Detonate_and_Analyze_File.yml:330]: should be exposed as playbook inputs

File: Packs/CrowdStrikeFalconX/Playbooks/playbook-FalconX_Detonate_and_Analyze_File.yml [hunk 3]
```diff
@@ -369,7 +385,49 @@ inputs:
   description: |-
     The operating system for which the alert was raised.
     accepted values are Windows, Linux, Android
-  playbookInputQuery:
+  playbookInputQuery: null
+- key: EnableTor
+  value:
+    simple: "false"
+  required: false
+  description: ""
```
Comment (yaakovi) [inline Packs/CrowdStrikeFalconX/Playbooks/playbook-FalconX_Detonate_and_Analyze_File.yml:393]: add descriptions to all inputs

File: Packs/CrowdStrikeFalconX/Playbooks/Detonate_File_-_CrowdStrike_Falcon_X.yml [hunk 4]
```diff
@@ -406,11 +406,20 @@ tasks:
...
       file:
         simple: ${inputs.File.EntryID}
       file_name:
         simple: ${inputs.File.Name}
+      action_script: {}
+      command_line: {}
+      document_password: {}
+      submit_name: {}
+      system_date: {}
+      system_time: {}
```
Comment (yaakovi) [inline Packs/CrowdStrikeFalconX/Playbooks/Detonate_File_-_CrowdStrike_Falcon_X.yml:422]: same as the other playbook (need to be exposed as playbook inputs)

**Review by yaakovi — APPROVED** [review]

**Pull request merged**